lib/aws/session_store/dynamo_db/configuration.rb

```diff
@@ -12,7 +12,7 @@
 # language governing permissions and limitations under the License.
 
 require 'yaml'
-require 'aws-sdk-v1'
+require 'aws-sdk'
 
 module AWS::SessionStore::DynamoDB
```

Comment:
That's reasonable enough. Like I said, there's probably a number of refactors we need to make or want to make before a full release of this. For example, I'm wondering if using `Aws::Record` wouldn't simplify a ton of this code, now that it exists.